proto/grp.proto

```diff
@@ -94,6 +94,10 @@ message Group {
   // Whether users are being managed by an external system using the SCIM API.
   // Disables in-app user management.
   bool external_user_management = 17;
+
+  // Maximum set of allowed capabilities that the user can assign to a user API
```

Comment:
Can you clarify this comment to make it more obvious that this restriction is specific to the currently authenticated user rather than something applicable to the group as a whole?

Reply:
Done